pkg/plugins/golang/declarative/v1/plugin.go

```diff
@@ -22,10 +22,10 @@ import (
 	cfgv3 "sigs.k8s.io/kubebuilder/v3/pkg/config/v3"
 	"sigs.k8s.io/kubebuilder/v3/pkg/model/resource"
 	"sigs.k8s.io/kubebuilder/v3/pkg/plugin"
-	"sigs.k8s.io/kubebuilder/v3/pkg/plugins"
+	"sigs.k8s.io/kubebuilder/v3/pkg/plugins/golang"
 )
 
-const pluginName = "declarative" + plugins.DefaultNameQualifier
+const pluginName = "declarative." + golang.DefaultGoNameQualifier
```

Comment:
Base plugins also require this dot unless you add it to the `golang.DefaultGoNameQualifier` const.